lib/repl.js

```diff
@@ -844,10 +844,49 @@ REPLServer.prototype.createContext = function() {
   });
 
   addBuiltinLibsToObject(context);
+  previewResults.call(this);
 
   return context;
 };
 
+function previewResults() {
```

Comment:
Do we really need this wrapping function? Also, it's confusing, since there is a variable with the same name.